virtcontainers/fc.go

```diff
@@ -57,6 +57,8 @@ const (
 var fcKernelParams = []Param{
 	// The boot source is the first partition of the first block device added
 	{"root", "/dev/vda1"},
+	{"rootflags", "data=ordered,errors=remount-ro ro"},
```

Comment:
Why `data=ordered`?
Do we really want `errors=remount-ro` - that means afaict, if we hit an error, mount the fs `ro`. But, we should not hit any errors should we - and, would we not rather abort if we did?

Reply:
That's a fair question, but we do already use these flags for qemu, but only for some architectures:

- https://github.com/kata-containers/runtime/blob/master/virtcontainers/qemu_amd64.go#L40
- https://github.com/kata-containers/runtime/blob/master/virtcontainers/qemu_arm64.go#L48

But we don't set `rootflags` for ppc64 nor for s390x. It does seem like we have an opportunity to make this all consistent?

/cc @devimc, @amshinde, @sboeuf.

Reply:
sure we can, unfortunately we still don't have a CI in place for all architectures hence we don't know if `ro` can break something. I think this change should be made by the architecture maintainer since he/she has the hw to test it.

cc @Pennyzct @nitkon @alicefr

Reply:
We should make this consistent. I would not expect to see any failures as there is no need for us to write to the rootfs (and we should not from a security point of view).  If we see any failure we should root cause and fix them.

@grahamwhaley Regarding the ordered as we do not write I do not think we need it, but I chose not to remove to keep consistency.

About the remount I was not sure if we needed it, but it felt safer.